knative-operator/pkg/common/openshift.go

```diff
@@ -33,6 +34,11 @@ func Mutate(ks *servingv1alpha1.KnativeServing, c client.Client) error {
 	return nil
 }
 
+func ingressClass(ks *servingv1alpha1.KnativeServing, c client.Client) error {
+	Configure(ks, "network", "ingress.class", "kourier.ingress.networking.knative.dev")
```

Comment:
Is that possible to use constants from knative.dev/serving ? instead of writing again

Reply:
I didn't realize those constants existed. But since they do, it might be a good idea to submit a separate PR that converts all the keys we pass to `Configure` to their upstream constants.

Reply:
`ingress.class` is here https://github.com/knative/serving/blob/master/pkg/network/network.go#L67

Reply:
Yes, Savita is correct. But it seems knative-operator still uses old code and we cannot find the value. https://github.com/openshift-knative/serverless-operator/tree/55d9f27c68286fb9be22fc03f3c3aa842db87054/knative-operator/vendor/knative.dev/pkg/network :cry:

Reply:
@jcrossley3 @markusthoemmes Do we have plan to use new code for serverless-operator ?

Reply:
Let's relay that for a separate PR.